charts/kueue/templates/webhook/service.yaml

```diff
@@ -12,6 +12,7 @@ spec:
   {{- end }}
   type: {{ .Values.webhookService.type }}
   selector:
+    control-plane: controller-manager
```

Comment:
no selectors

Reply:
This was removed here: https://github.com/kubernetes-sigs/kueue/pull/1695/files#diff-cf031362823433715af967430243e9d5edc133805d6f62b8bb07f4bd359710dc

Reply:
Removed